projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-roles-endpoint.php

```diff
@@ -124,7 +124,7 @@ public function callback( $path = '', $blog_id = 0 ) {
 			return new WP_Error( 'unauthorized', 'User cannot view roles for specified site', 403 );
 		}
 
-		if ( method_exists( $wp_roles, 'get_names' ) ) {
+		if ( $wp_roles instanceof WP_Roles ) {
```

Comment:
Maybe we could be more defensive here and also check if `$wp_roles` is actually an array next?

Reply:
Thanks for reviewing, I addressed it in 6540dbd